Sources/SwiftProtobuf/ProtobufUtility.swift

```diff
@@ -27,11 +27,11 @@ public func ==(lhs: [[UInt8]], rhs: [[UInt8]]) -> Bool {
     return true
 }
 
-public func !=(lhs: [[UInt8]], rhs: [[UInt8]]) -> Bool {
+func !=(lhs: [[UInt8]], rhs: [[UInt8]]) -> Bool {
     return !(lhs == rhs)
 }
 
-public func ==<T: Equatable>(lhs: Dictionary<T, [UInt8]>, rhs: Dictionary<T, [UInt8]>) -> Bool {
+func ==<T: Equatable>(lhs: Dictionary<T, [UInt8]>, rhs: Dictionary<T, [UInt8]>) -> Bool {
```

Comment:
Ditto here:  `map<T, bytes>` now gets rendered as `Dictionary<T, Data>` instead of `Dictionary<T,[UInt8]>`, so these helpers are probably no longer needed at all.